Khan/wonder-blocks
# Add ariaLabel prop to Banner

## Summary:
There are [cases](https://github.com/Khan/webapp/blob/master/services/static/javascript/site-banners/shared/base-banner.jsx#L58
) of legacy banners in webapp that have
screen reader only headers. We can add this functionality
in WB Banners by adding an aria label prop.

https://user-images.githubusercontent.com/13231763/200427306-d89ea648-d45d-4349-8304-334c02982597.mov


Issue: none

## Test plan:
Storybook
- Go to the "With Dismissal" story.
- Dismiss the banner.
- Turn on the screenreader.
- Press the button to make the banner come back.
- Confirm that the screenreader reads that aria label,
  and then the rest of the contents of the banner.
